### DISCUSSION TITLE
How to set accessibility (keyboard navigation) for progress bar (`Slider Video`)?

### DISCUSSION THREAD

**Discussion by pietrop** [vidstack/player#692] [Q&A]
_non urgent question_

When getting to the progress bar ([Slider Video](https://www.vidstack.io/docs/player/components/ui/slider-video/))via Tab navigation in the docs I am not able to move arrows left and right to change current time.

I did notice that it does work in the [Svelte demo app](https://www.vidstack.io/)

And was trying to figure out if it's because of the [Gesture](https://www.vidstack.io/docs/player/react/components/ui/gesture/)?

[vidstack/player/apps/site/src/pages/index/PlayerGestures.svelte](https://github.com/vidstack/player/blob/bf0982c9a267d648762eb43aa84006215ad0c440/apps/site/src/pages/index/PlayerGestures.svelte)

Or some other config on the `PlayerTimeSlider`?

https://github.com/vidstack/player/blob/bf0982c9a267d648762eb43aa84006215ad0c440/apps/site/src/pages/index/PlayerTimeSlider.svelte#L15

**Comment by mihar-22** [comment]
That's strange, I just tested it and it worked. Specific browser issue?

**Reply by pietrop** [reply]
I figured it out, it doesn't seem to work if the video is playing (tab to the progress bar and use left or right arrow while video is playing doesn't seem to be doing anything). But It does work if it's paused. Not sure if it's a feature or a 🐛 ?

**Reply by mihar-22** [reply]
Interesting. I don't know how keyboard input is actually working since it's a custom element. I wonder if the browser is inferring from ARIA labels and adding primitive keyboard controls? 

Do note we haven't added keyboard support yet so not too concerned about this.

**Reply by pietrop** [reply]
ah, ok, cool. Good to know, thanks!
